javalin/src/main/java/io/javalin/http/Context.kt

```diff
@@ -388,11 +388,22 @@ open class Context(@JvmField val req: HttpServletRequest, @JvmField val res: Htt
 
     /** Sets the response status code and redirects to the specified location. */
     @JvmOverloads
-    fun redirect(location: String, httpStatusCode: Int = HttpServletResponse.SC_MOVED_TEMPORARILY) {
+    fun redirect(location: String, httpStatusCode: Int = HttpCode.TEMPORARY_REDIRECT.status) {
         res.setHeader(Header.LOCATION, location)
         status(httpStatusCode)
+
         if (handlerType == HandlerType.BEFORE) {
```

Comment:
This feels like a bodge job from before so I am just assuming that my additions here are correct.